release/air_tests/air_benchmarks/workloads/tensorflow_benchmark.py

```diff
@@ -82,6 +82,7 @@ def _handle(self, logs: dict, when: str = None):
         epochs=epochs,
         steps_per_epoch=steps_per_epoch,
         callbacks=callbacks,
+        verbose=0,  # Disables progress bar in remote actors.
```

Comment:
```suggestion
        verbose=2,  # Disables progress bar in remote actors.
```

can we set this to 2 instead

```
'auto', 0, 1, or 2. Verbosity mode. 0 = silent, 1 = progress bar, 2 = one line per epoch. 'auto' defaults to 1 for most cases, but 2 when used with ParameterServerStrategy. Note that the progress bar is not particularly useful when logged to a file, so verbose=2 is recommended when not running interactively (eg, in a production environment).
```

https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit

Reply:
sounds good, thanks!